implementations/rust/ockam/ockam_api/src/nodes/service/secure_channel.rs

```diff
@@ -90,8 +92,18 @@ impl NodeManager {
         authorized_identifiers: Option<Vec<IdentityIdentifier>>,
         credential_exchange_mode: CredentialExchangeMode,
         timeout: Option<Duration>,
+        identity_ctx: Option<(CowStr<'_>, &Context)>,
```

Comment:
In the context of this function, you might want to rename this to `identity_name` as it could get easily confused with the actual identity that you build in the next line.